rotkehlchen/chain/ethereum/modules/aave/v2/accountant.py

```diff
@@ -0,0 +1,92 @@
+from collections import defaultdict
+from typing import TYPE_CHECKING
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry, get_tx_event_type_identifier
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.ethereum.accounting.interfaces import ModuleAccountantInterface
+from rotkehlchen.chain.ethereum.accounting.structures import TxEventSettings
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.misc import ZERO
+from rotkehlchen.fval import FVal
+from rotkehlchen.types import ChecksumEvmAddress
+
+from ..constants import CPT_AAVE_V2
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.pot import AccountingPot
+
+
+class Aavev2Accountant(ModuleAccountantInterface):
+    """Accountant for Aave v2 protocol"""
+    def reset(self) -> None:
+        self.assets_borrowed: dict[tuple[ChecksumEvmAddress, Asset], FVal] = defaultdict(FVal)
+
+    def _process_borrow(
+            self,
+            pot: 'AccountingPot',  # pylint: disable=unused-argument
+            event: HistoryBaseEntry,
+            other_events: list[HistoryBaseEntry],  # pylint: disable=unused-argument
+    ) -> None:
+        assert event.location_label is not None, 'Aave v2 borrow event must have a location label'
+        self.assets_borrowed[(string_to_evm_address(event.location_label), event.asset)] += event.balance.amount  # noqa: E501
+
+    def _process_payback(
+            self,
+            pot: 'AccountingPot',
+            event: HistoryBaseEntry,
+            other_events: list[HistoryBaseEntry],  # pylint: disable=unused-argument
+    ) -> None:
+        assert event.location_label is not None, 'Aave v2 payback event must have a location label'
```

Comment:
We shouldn't be using assertions here since it can break the app and we can't recover from it. Here and in the other method

Reply:
How would you do it then? `event.location_label` is guaranteed to be not None here and the assertion is just for typing purposes

Reply:
Okay